### PR TITLE
[zh-CN]: add translation for `Navigation`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

[Navigation](https://developer.mozilla.org/en-US/docs/Web/API/Navigation)

### Related issues and pull requests

part of the https://github.com/mdn/translated-content/issues/24188

### CODE REVIEW

File: files/zh-cn/web/api/navigation/index.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+title: Navigation
+slug: Web/API/Navigation
+l10n:
+  sourceCommit: 132d7ff76c89ed913b27eb85fc353adc3eb00e06
+---
+
+{{APIRef("Navigation API")}}{{SeeCompatTable}}
+
+{{domxref("Navigation API", "Navigation API", "", "nocode")}} 的 **`Navigation`** 接口允许在一个核心位置控制当前 `window` 的所有导航操作，包括以编程方式启动导航、检查导航历史记录条目以及在导航发生时管理导航。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigation/index.md:10]: 可以翻译一下 API

```suggestion
{{domxref("Navigation API", "导航 API", "", "nocode")}} 的 **`Navigation`** 接口允许在一个核心位置控制当前 `window` 的所有导航操作，包括以编程方式启动导航、检查导航历史记录条目以及在导航发生时管理导航。
```

File: files/zh-cn/web/api/navigation/index.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+它是通过 {{domxref("Window.navigation")}} 属性访问的。
+
+Navigation API 仅公开在当前浏览上下文中创建的与当前页面同源的历史记录条目（例如，不是嵌入的 {{htmlelement("iframe")}} 内的导航或跨源导航），为你的应用提供所有先前历史记录条目的准确列表。与旧版 {{domxref("History API", "History API", "", "nocode")}} 相比，这使得遍历历史记录变得不那么脆弱。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigation/index.md:14]:
```suggestion
导航 API 仅公开在当前浏览上下文中创建的与当前页面同源的历史记录条目（例如，不是嵌入的 {{htmlelement("iframe")}} 内的导航或跨源导航），为你的应用提供所有先前历史记录条目的准确列表。与旧版{{domxref("History API", "历史记录 API", "", "nocode")}} 相比，这使得遍历历史记录变得不那么脆弱。
```

File: files/zh-cn/web/api/navigation/index.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+{{InheritanceDiagram}}
+
+## 实例属性
+
+_从其父级接口 {{DOMxRef("EventTarget")}} 继承属性。_
```
Comment (yin1999) [inline files/zh-cn/web/api/navigation/index.md:20]:
```suggestion
_从其父接口 {{DOMxRef("EventTarget")}} 继承属性。_
```

File: files/zh-cn/web/api/navigation/index.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+- {{domxref("Navigation.canGoForward", "canGoForward")}} {{ReadOnlyInline}} {{Experimental_Inline}}
+  - : 如果可以在导航历史记录中向前导航（即 {{domxref("Navigation.currentEntry", "currentEntry")}} 不是历史记录条目列表中的最后一个），则返回 `true`，如果不能，则返回 `false`。
+- {{domxref("Navigation.currentEntry", "currentEntry")}} {{ReadOnlyInline}} {{Experimental_Inline}}
+  - : 返回一个 {{domxref("NavigationHistoryEntry")}} 对象，表示用户当前导航到的历史记录。
+- {{domxref("Navigation.transition", "transition")}} {{ReadOnlyInline}} {{Experimental_Inline}}
+  - : 返回一个 {{domxref("NavigationTransition")}} 对象，表示正在进行的导航的状态，可用于跟踪导航。如果当前没有正在进行的导航，则返回 `null`。
+
+## 实例方法
+
+_从其父级接口 {{DOMxRef("EventTarget")}} 继承方法。_
```
Comment (yin1999) [inline files/zh-cn/web/api/navigation/index.md:35]:
```suggestion
_从其父接口 {{DOMxRef("EventTarget")}} 继承方法。_
```

File: files/zh-cn/web/api/navigation/index.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [现代客户端路由：Navigation API](https://developer.chrome.google.cn/docs/web-platform/navigation-api/)
+- [Navigation API 说明](https://github.com/WICG/navigation-api/blob/main/README.md)
+- Domenic Denicola 的 [Navigation API 在线示例](https://gigantic-honored-octagon.glitch.me/)
```
Comment (yin1999) [inline files/zh-cn/web/api/navigation/index.md:130]: 这里也可以翻译一下

Comment (skyclouds2001) [inline files/zh-cn/web/api/navigation/index.md:130]:
```suggestion
- [现代客户端路由：导航 API](https://developer.chrome.google.cn/docs/web-platform/navigation-api/)
- [导航 API 说明](https://github.com/WICG/navigation-api/blob/main/README.md)
- Domenic Denicola 的 [导航 API 在线示例](https://gigantic-honored-octagon.glitch.me/)
```

**Review by yin1999 — APPROVED** [review]
Thank you @skyclouds2001

**Pull request merged**